firebase-database/src/main/java/com/google/firebase/database/FirebaseDatabase.java

```diff
@@ -38,7 +38,7 @@
 @PublicApi
 public class FirebaseDatabase {
 
-  private static final String SDK_VERSION = "3.0.0";
+  private static final String SDK_VERSION = BuildConfig.VERSION_NAME;
```

Comment:
This almost seems like it would be better reviewed by someone on the acore team. How does the version get set in the gradle configuration?

https://github.com/firebase/firebase-android-sdk/blob/40439c3b1dbd53bb96014652a453d44e0a840a49/firebase-database/firebase-database.gradle#L34

Reply:
I assigned this to Vlad. FWIW, we already use this version here: https://github.com/firebase/firebase-android-sdk/blob/master/firebase-database/src/main/java/com/google/firebase/database/DatabaseRegistrar.java#L43

Reply:
Ah. In that case, LGTM.